### PR TITLE
Add github templates

### PR DESCRIPTION
**Pull request by blink1073**
Add github templates based on the ones in [JupyterHub](https://github.com/jupyterhub/jupyterhub/tree/56601d93c3cd9c1e8fd599886877c14f54bf4a46/.github).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+
+- OS: [e.g. iOS]
```
Comment (afshin) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Let's use an OS we support as the example ;-)

File: .github/ISSUE_TEMPLATE/installation-and-configuration-issues.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+name: Installation and configuration issues
+about: Installation and configuration assistance
+---
+
+If you are having issues with installation or configuration, you may ask for help on the JupyterLab gitter channel or file an issue here.
```
Comment (afshin) [inline .github/ISSUE_TEMPLATE/installation-and-configuration-issues.md]: [JupyterLab Gitter channel](https://gitter.im/jupyterlab/jupyterlab)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (blink1073) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Hahahaha, d'oh.

File: .github/ISSUE_TEMPLATE/installation-and-configuration-issues.md (hunk 2 above)
Comment (afshin) [reply .github/ISSUE_TEMPLATE/installation-and-configuration-issues.md]: i.e. let's make that a link.

**Comment by vidartf** [comment]
I'd love for it to include a note about checking the browser console, and including any relevant parts!

**Comment by vidartf** [comment]
PS: Useful template for including long logs:

```html
<details><summary>Short description</summary>
<pre>
Paste your log here!
</pre>
</details>
```

**Comment by blink1073** [comment]
Great suggestions, @vidartf!  Updated.

**Pull request merged**